shared/js/background/pixels.js

```diff
@@ -20,8 +21,11 @@ export function sendPixelRequest (pixelName, params = {}) {
         return
     }
 
+    const browserName = getBrowserName() || 'unknown'
+
     const randomNum = Math.ceil(Math.random() * 1e7)
     const searchParams = new URLSearchParams(Object.entries(params))
-    const url = getURL(pixelName) + `?${randomNum}&${searchParams.toString()}`
+    const url = getURL(`${pixelName}_extension_${browserName}`) +
```

Comment:
This seems to be a different naming convention to on the browser site report pixel, which only sends browser name (does not send `extension`), but does not change `firefox` to `moz`. Can we align the behavior for all pixels? We might need to sync with @GioSensation if this will affect current autofill pixels.

Reply:
Pixels are not sent for DDG extension users on Firefox, so I don't think `firefox` vs `moz` is an issue.

My preference is to stick with the "_extension_BROWSERNAME" suffix convention @GioSensation set up for the autofill pixels here, to avoid messing with those pixels. I also checked with @englehardt  and @lrjball that the suffix works for them.

If you're against that suffix, what are you asking for instead?

Reply:
This does affect autofill pixels, but the number of panels to update is relatively low. The `extension_BROWSERNAME` convention makes it easier to distinguish from mobile, so that you can group all mobile and all extensions separately. It's not critical, but makes it slightly easier. I'm not entirely sure what's the goal, as @kzar was asking.